src/config/manager.test.ts

```
@@ -0,0 +1,68 @@
+        expect(() => cm.validate()).not.toThrow();
+    });
+
+    it('throws when MCP server configs missing', () => {
+        const bad: AgentConfig = clone(baseConfig);
+        bad.mcpServers = {};
+        expect(() => new ConfigManager(bad)).toThrow(/No MCP server configurations provided/);
+    });
+
+    it('throws when LLM config is missing', () => {
+        const bad: AgentConfig = clone(baseConfig);
+        delete bad.llm;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)